Managers/MetaMan.cpp

```diff
@@ -1035,29 +1035,19 @@ int MetaMan::TotalScenePresets(std::list<Scene *> *pScenes)
 //////////////////////////////////////////////////////////////////////////////////////////
 // Description:     Yields a set of randomly selected Scene presets for a new game.
 
-int MetaMan::SelectScenePresets(float gameSize, int playerCount, list<Scene *> *pSelected)
+int MetaMan::SelectScenePresets(int gameSize, list<Scene *> *pSelected)
 {
     // Get the list of ALL eligible read-in Scene presets
     list<Scene *> scenePresets;
     TotalScenePresets(&scenePresets);
 
-    // How many scenes the game should end up with, according to the specified game size.
```

Comment:
I assume this change is cause the slider now directly defines the number of players?

Reply:
This slider now directly defines the number of sites in the game.
I know, it's a little messy...
All the removed stuff from this function now happens outside of it, at MetagameGUI::UpdateGameSizeLabels().

Reply:
This function, SelectScenePresets, used to set the minimum and maximum values for the slider.
that only needs to be done at MetagameGUI::UpdateGameSizeLabels() anyway, so i moved that part out to there.
the second part of this function is to pick scenes for the game,
that only matters at MetaMan::NewGame, and it's the only thing remaining in the function, so now it is only called from NewGame.

Reply:
Okay, that all sounds good to me!